metasyn/demo/dataset.py

```diff
@@ -146,56 +183,92 @@ def name(self):
-                                        np.random.randn(n_rows),
-                                        dtype=getattr(pl, f"Float{float_val}")))
-
-        all_series.append(pl.Series("pl.Date", [date(2024, 9, 4) + timedelta(days=i)
-                                                for i in range(n_rows)],
-                                    dtype=pl.Date))
-        all_series.append(pl.Series("pl.Datetime",
-                                    [datetime(2024, 9, 4, 12, 30, 12)
-                                        + timedelta(hours=i, minutes=i*2, seconds=i*3)
-                                        for i in range(n_rows)],
-                                    dtype=pl.Datetime))
-        all_series.append(pl.Series("pl.Time",
-                                    [time(3+i//20, 6+i//12, 12+i//35) for i in range(n_rows)],
-                                    dtype=pl.Time))
-        all_series.append(pl.Series("pl.String",
-                                    np.random.choice(list(string.printable), size=n_rows),
-                                    dtype=pl.String))
-        all_series.append(pl.Series("pl.Utf8",
-                                    np.random.choice(list(string.printable), size=n_rows),
-                                    dtype=pl.Utf8))
-        all_series.append(pl.Series("pl.Categorical",
-                                    np.random.choice(list(string.ascii_uppercase[:5]), size=n_rows),
-                                    dtype=pl.Categorical))
-        all_series.append(pl.Series("pl.Boolean",
-                                    np.random.choice([True, False], size=n_rows),
-                                    dtype=pl.Boolean))
+            all_series.append(
+                pl.Series(
+                    f"pl.Float{float_val}",
+                    np.random.randn(n_rows),
+                    dtype=getattr(pl, f"Float{float_val}"),
+                )
+            )
+
+        all_series.append(
+            pl.Series(
+                "pl.Date",
+                [date(2024, 9, 4) + timedelta(days=i) for i in range(n_rows)],
+                dtype=pl.Date,
+            )
+        )
+        all_series.append(
+            pl.Series(
+                "pl.Datetime",
+                [
+                    datetime(2024, 9, 4, 12, 30, 12)
+                    + timedelta(hours=i, minutes=i * 2, seconds=i * 3)
+                    for i in range(n_rows)
+                ],
+                dtype=pl.Datetime,
+            )
+        )
+        all_series.append(
+            pl.Series(
+                "pl.Time",
+                [time(3 + i // 20, 6 + i // 12, 12 + i // 35) for i in range(n_rows)],
+                dtype=pl.Time,
+            )
+        )
+        all_series.append(
+            pl.Series(
+                "pl.String", np.random.choice(list(string.printable), size=n_rows), dtype=pl.String
+            )
+        )
+        all_series.append(
```

Comment:
I presume this is all just automatic reformatting, so I haven't checked on differences.

Reply:
Correct, ruff format